sdk/java/pulumi/src/main/java/com/pulumi/automation/DestroyOptions.java

```diff
@@ -29,6 +38,15 @@ public static Builder builder() {
         return new Builder();
     }
 
+    /**
+     * The list of specified targets.
+     *
+     * @return The specified targets
+     */
+    public List<String> targets() {
```

Comment:
This is exposed on the base `UpdateOptions`: 

https://github.com/pulumi/pulumi-java/blob/e8009acfe682371b1d6704efef3d5ea1545e601e/sdk/java/pulumi/src/main/java/com/pulumi/automation/UpdateOptions.java#L87-L95

But I guess I missed passing the CLI flags if it's set?